salt/_pillar/velum.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+'''
+A module that adds data to the Pillar structure retrieved by an http request made to Velum
```

Comment:
Nitpicking: can you add a line explaining that results are also cached into salt's cache to ensure they are accessible even when velum is temporarily down?